tqdm/_utils.py

```diff
@@ -1,5 +1,12 @@
 import os
 import subprocess
+from platform import system as _curos
+CUR_OS = _curos()
+IS_WIN = CUR_OS in ['Windows', 'cli']
+IS_NIX = (not IS_WIN) and any(
+    CUR_OS.startswith(i) for i in
+    ['CYGWIN', 'MSYS', 'Linux', 'Darwin', 'SunOS', 'FreeBSD'])
```

Comment:
Wouldn't want `lower` to encourage consistency. `platform.system()` should never equal, for example, `windows`. Also we can't assume all systems which we fail to detect as windows systems are unix. I'd be happier to add "if neither nix nor win, print warning"